metis-enrichment/metis-enrichment-common/src/main/java/eu/europeana/enrichment/api/internal/ReferenceTermContext.java

```diff
@@ -17,22 +20,11 @@ public ReferenceTermContext(URL reference, Set<FieldType> fieldTypes) {
   }
 
   @Override
-  public List<EntityType> getCandidateTypes() {
-    return fieldTypes.stream().map(FieldType::getEntityType).collect(Collectors.toList());
+  public Set<EntityType> getCandidateTypes() {
+    return fieldTypes.stream().map(FieldType::getEntityType).collect(Collectors.toSet());
   }
 
-  @Override
```

Comment:
I think equals and hashcode should be declared here too, preferably with using the new fields or just by using super